stable/sidecar-terminator/templates/clusterrolebinding.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ include "sidecar-terminator.fullname" . }}
+  labels:
+    {{- include "sidecar-terminator.labels" . | nindent 4 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ include "sidecar-terminator.fullname" . }}
+subjects:
+- kind: ServiceAccount
+  name: {{ include "sidecar-terminator.fullname" . }}
+  namespace: {{ .Values.namespace }}
```

Oh and this is where you would put a 
```suggestion
  namespace: {{ .Release.Namespace }}
```